libs/vertexai/tests/unit_tests/test_chat_models.py

```diff
@@ -1067,3 +1068,49 @@ def test_init_client_with_custom_api() -> None:
         transport = mock_prediction_service.call_args.kwargs["transport"]
         assert client_options.api_endpoint == "https://example.com"
         assert transport == "rest"
+
+@pytest.mark.requires("anthropic")
```

Comment:
I'm not sure we support requires marker. Probably, it's easier just to add it to the test group (we need to install it anyway to run the whole test suite).